lua/lspconfig/server_configurations/ruff_lsp.lua

```diff
@@ -26,16 +27,16 @@ Extra CLI arguments for `ruff` can be provided via
 
 ```lua
 require'lspconfig'.ruff_lsp.setup{
-  settings = {
-    ruff_lsp = {
+  init_options = {
+    settings = {
       -- Any extra CLI arguments for `ruff` go here.
-      args = {},
+      args = { '--line-length=120' },
```

Comment:
What's the reasoning for this?

Reply:
(It was meant to demonstrate the argument-passing style, but perhaps that's out-of-character for these docs.)

Reply:
Nah I was too quick to read and didn't notice it was for the docs (I thought you updated the server config). So I guess it would've been fine.